godot-bindings/src/header_gen.rs

```diff
@@ -5,8 +5,56 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-use std::env;
+use regex::Regex;
 use std::path::Path;
+use std::{env, fs};
+
+pub(crate) fn patch_c_header(in_h_path: &Path, out_h_path: &Path) {
```

Comment:
Could you keep this in the old place for now, and maybe we can move it later?

Right now it's impossible for me to review what really changed; on top of that there are subtle formatting changes.

Reply:
I moved it & removed `GODOT4_GDEXTENSION_HEADERS` feature (as mentioned in commit). Bringing it back (after someone presents solid usecase for it) should be rather trivial :thinking:.